github/tests/Exceptions.py

```diff
@@ -123,3 +123,12 @@ def exceed():
                 g.get_user("jacquev6")
 
         self.assertRaises(github.RateLimitExceededException, exceed)
+
+    def testAuthenticatedRateLimitExceeded(self):
+
+        def exceed():
+            for i in range(100):
```

Comment:
Interesting, how fast does the test run? I am a little worried about the running time because of the loop here

Reply:
```
└─[0] <git:(fixed_ratelimit c43acd5) > python -m github.tests SpecificExceptions.testAuthenticatedRateLimitExceeded
.
----------------------------------------------------------------------
Ran 1 test in 0.052s

OK
```